pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -177,13 +176,6 @@ func (uc *UpgradeCluster) getClusterNodeStatus(az armhelpers.AKSEngineClient, re
 				continue
 			}
 
-			// Skip the VM upgrade validation for managed clusters as it only applies to aks-engine version support.
```

Comment:
removeing this validation as it is redundant with the cmd validation, and ignored for hosted master clusters

Reply:
this comment isn't relevant anymore is it?

Reply:
nope resolving